### PR TITLE
Click element to keep its tooltip on the screen, click away to dismiss

### PR DESCRIPTION
**Pull request by wbazant**
Demo at https://wbazant.github.io/react-tooltip-demo/

In our project we want the user to:
1. Hover over a table column, show tooltip
2. Click column, keep the tooltip on the screen
3. Access link inside tooltip or
4. Click outside tooltip to dismiss

Don't merge this PR as it is because I probably didn't think about everything, also it's not very elegant because half the implementation is outside (see demo code - keeping the state of frozen outside, toggle frozen when clicking the component, and on click outside tooltip also toggle frozen).

This solution will work for our project and I want to show it to you, and maybe inspire you to come up with a better way to add this functionality :)

### CODE REVIEW

**Comment by wwayne** [comment]
That's pretty useful I think. I was planning to add a feature so that user can move mouse to the tooltip itself(current solution is awful).
And yeah, clicking to keep the tooltip is also a good idea, I can try to implement this later, let's keep this PR open for the moment, thanks for your explain and demo : )

**Comment by thedaviddias** [comment]
Can someone try to correct these conflicts? @wbazant @alfonsomunozpomer

**Comment by wbazant** [comment]
@thedaviddias thank you for your interest in this PR! :) I no longer use the component and don't want to commit time to test the change, rerun a build, and in general do a proper job.

Feel free to hm, fix the change and PR against my branch, I will merge, and merge here. Or squash my commits, add the change yourself, and tell me to close the PR.

File: src/index.js [hunk 1]
```diff
@@ -245,8 +268,8 @@ class ReactTooltip extends Component {
     }
 
     this.setState({
-      placeholder,
-      isEmptyTip,
+      originTooltip: originTooltip,
+      isMultiline: isMultiline,
```
Comment (reinaldosimoes) [inline src/index.js:272]: Just a small tweak, everything else looks good

```suggestion
      originTooltip,
      isMultiline,
```

**Comment by wbazant** [comment]
I agree these are better, thank you for reviewing! If you have push access
to the branch do change it and merge. I only have my small internet device
for the next month or so, and no longer depend on the component.

W dniu niedziela, 13 grudnia 2020 Reinaldo Simoes <notifications@github.com>
napisał(a):

**Comment by danielbarion** [comment]
Hi guys, can someone take a look on that PR, please?

**Comment by wbazant** [comment]
FYI all, I'm gonna unsubscribe myself from this thread now. Leaving it to @danielbarion to take care of any development that needs to happen to get this feature in

**Comment by danielbarion** [comment]
Closing due to V5

**Pull request closed without merging**